From b9b94712847f8fa6ca8dc03422ba9e0332b6997d Mon Sep 17 00:00:00 2001
From: Robert Marko <robimarko@gmail.com>
Date: Thu, 13 May 2021 14:54:43 +0200
Subject: [PATCH] soc: qcom: Make QMI Helpers user selectable

Wireless backports for ath11k have a different logic
then upstream as they depend on QMI helpers instead
of selecting them.

So, to make it possible to select QMI helpers
through KConfig add text to tristate.

Signed-off-by: Robert Marko <robimarko@gmail.com>
---
 drivers/soc/qcom/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/soc/qcom/Kconfig
+++ b/drivers/soc/qcom/Kconfig
@@ -92,7 +92,7 @@ config QCOM_PDR_HELPERS
 	select QCOM_QMI_HELPERS
 
 config QCOM_QMI_HELPERS
-	tristate
+	tristate "Qualcomm QMI helpers"
 	depends on NET
 
 config QCOM_RMTFS_MEM
